src/OSPSuite.Infrastructure/Services/DataRepositoryTask.cs

```diff
@@ -99,16 +99,35 @@ private string idOfCloneFor(DataColumn column)
          return _idMap[column.Id];
       }
 
-      public IEnumerable<DataTable> ToDataTable(IEnumerable<DataColumn> dataColumns, bool formatOutput = false, bool useDisplayUnit = true)
+      public IEnumerable<DataTable> ToDataTable(IEnumerable<DataColumn> dataColumns, bool formatOutput = false, bool useDisplayUnit = true, bool forceColumnTypeAsObject=false)
       {
-         return ToDataTable(dataColumns, c => c.Name, c => c.Dimension, formatOutput, useDisplayUnit);
+         return ToDataTable(dataColumns, c => c.Name, c => c.Dimension, formatOutput, useDisplayUnit, forceColumnTypeAsObject);
       }
 
-      public IEnumerable<DataTable> ToDataTable(IEnumerable<DataColumn> dataColumns, Func<DataColumn, string> columnNameRetriever, Func<DataColumn, IDimension> dimensionRetrieverFunc, bool formatOutput = false, bool useDisplayUnit = true)
+      public IEnumerable<DataTable> ToDataTable(
+         IEnumerable<DataColumn> dataColumns, 
+         Func<DataColumn, string> columnNameRetriever, 
+         Func<DataColumn, IDimension> dimensionRetrieverFunc, 
+         bool formatOutput = false, 
+         bool useDisplayUnit = true,
+         bool forceColumnTypeToObject = false
+         )
       {
          var allColumns = allColumnsWithRelatedColumnsFrom(dataColumns);
          var cacheName = retrieveUniqueNamesForTables(allColumns);
-         return cacheName.KeyValues.Select(keyValuePair => createTableFor(keyValuePair.Value, keyValuePair.Key, allColumns, columnNameRetriever, dimensionRetrieverFunc, formatOutput, useDisplayUnit));
+         var valueFormatter = valueFormatterFor(formatOutput);
+         var columnType = forceColumnTypeToObject ? typeof(object) : formatOutput ? typeof(string) : typeof(float);
```

Comment:
If the datatable is used for binding, the type needs to be object otherwise devexpress believe it knows better and starts to convert 6,5 in 65